src/unix/linux_like/linux/gnu/mod.rs

```diff
@@ -1399,6 +1399,9 @@ extern "C" {
         envp: *const *mut c_char,
         flags: ::c_int,
     ) -> ::c_int;
+
+    // Added in `glibc` 2.34
+    pub fn close_range(first: ::c_uint, last: ::c_uint, flags: ::c_int) -> ::c_int;
```

Comment:
This `flags` argument, is of type `unsigned int`, according to the signature given in the man page, but

```sh
$ cd libc-test && cargo t
```
would complain it should have type `int` to me

Reply:
glibc version difference? I'd prefer to declare a newer one.

Reply:
> glibc version difference?

Not sure about this, let me investigate it a little bit, my glibc is 2.37

```sh
$ ldd --version
ldd (GNU libc) 2.37
Copyright (C) 2023 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
Written by Roland McGrath and Ulrich Drepper.
```

Reply:
Just give it a check on glibc

* 2.34
* 2.35
* 2.36

The `flag` argument is of type int, so I guess this is a mistake of the manual

Reply:
Seems so, thank you for investigating!